packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -117,14 +145,15 @@ describe('SystemRulesTable component', () => {
                 loading={ false }
                 system={ system }
                 itemsPerPage={ 50 }
+                columns={ columns }
             />
         );
         const instance = wrapper.instance();
         await instance.setInitialCurrentRows();
         await instance.updateFilter(false, [ 'high', 'medium' ], []);
         expect(wrapper.state('currentRows').length / 2).toEqual(50);
         wrapper.state('currentRows').forEach((row, i) => {
-            if (row.hasOwnProperty('parent')) {
+            if (Object.prototype.hasOwnProperty.call(row, 'parent')) {
```

Comment:
Curious why this change is needed? calling `hasOwnProperty` on `row` should be the same.

Reply:
New `eslint` requirement for the whole project :man_shrugging: